atest/testdata/running/skip/skip_with_template.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Library            skiplib.py
+
+*** Variables ***
+${TEST_OR_TASK}    Test
```

Comment:
Neither this variable nor the library above don't seem to be used for anything and can be removed.